src/Lean/Elab/Tactic/Location.lean

```diff
@@ -21,7 +21,8 @@ inductive Location where
 Recall that
 ```
 syntax locationWildcard := "*"
-syntax locationHyp      := (colGt term:max)+ ("⊢" <|> "|-")?
+syntax locationTurnstile := ("|-" <|> "⊢")
```

Comment:
Doesn't this make it a token? (Maybe I'm confused)

Reply:
Oh, sorry, good catch; I forgot to update this comment. (The actual declaration in `Init/Tactics.lean` is correct, though.)

Reply:
Oh, it's a comment! My bad. (Easy to miss when looking at the side)